src/styles/primary.js

```diff
@@ -11,3 +11,6 @@ export const darkBlue = '#1F1F27';
 
 // Social media
 export const facebookBlue ='#3b5998';
+
+// Campsite Icons
+export const campsiteIcon = '#228b22';
```

Comment:
Named this for what it's doing instead of what color it is, so if we decide to change the color of the icon we don't have to also change the name.